### ISSUE TITLE
did someone make a good bot or not?

### ISSUE THREAD

**Issue by Nyantad** [Pbatch/ClashRoyaleBuildABot#22]
I was just wondering if someone had made a BOT smarter than PETE or not?

**Comment by Pbatch** [comment]
Not at the minute! If you have any luck, please post your code. 

I have a few more changes I want to make to the state generation, then I will have a go at making PeteBot better.
There should be a new version out in a few weeks :)

**Comment by Nyantad** [comment]
okay I'll keep that in mind, thanks for the answer

**Comment by Pbatch** [comment]
@Nyantad Hello again! I'm working on a new branch, with a custom score function for each card in the standard deck :) 
https://github.com/Pbatch/ClashRoyaleBuildABot/pull/26

Give it a try and let me know what you think/ what it is missing. I think you'll find it's much more powerful than PeteBot. 

I'll put a video in the Wiki soon explaining the choices it is making.

@SergeyPotapov01 your software looks like it's very good at navigating the game interface. Maybe you can help accelerate that side of things for this project!

**Comment by Nyantad** [comment]
Yes for the bot I will look at it soon, but on the other hand I looked at the BOT of @SergeyPotapov01 and indeed it would be a good idea to make a collaboration and to put several things in common like that the development could greatly develop !

**Comment by SergeyPotapov01** [comment]
> Да на бота скоро посмотрю, а с другой стороны посмотрел на БОТа@SergeyPotapov01и действительно, было бы неплохо сделать совместную работу и объединить несколько вещей, чтобы развитие могло значительно развиваться!


I was thinking about how to combine these two bots. But I abandoned this idea. It seemed to me that this bot works only for a small part of the cards, and in order to increase the possibilities for playing with all the cards, it will take tens of hours. I actually stopped the development of AI for the competitive part and took up the microfeatures that users need to automate the gameplay
